libs/ai-endpoints/tests/integration_tests/test_register_model.py

```diff
@@ -16,29 +16,34 @@
 # you will have to find the new ones from https://api.nvcf.nvidia.com/v2/nvcf/functions
 #
 @pytest.mark.parametrize(
-    "client, id, endpoint",
+    "client, id, endpoint, model_type",
     [
         (
             ChatNVIDIA,
             "meta/llama3-8b-instruct",
             "https://api.nvcf.nvidia.com/v2/nvcf/pexec/functions/a5a3ad64-ec2c-4bfc-8ef7-5636f26630fe",
+            "chat",
         ),
         (
             NVIDIAEmbeddings,
             "NV-Embed-QA",
             "https://api.nvcf.nvidia.com/v2/nvcf/pexec/functions/09c64e32-2b65-4892-a285-2f585408d118",
+            "embedding",
         ),
         (
             NVIDIARerank,
             "nv-rerank-qa-mistral-4b:1",
             "https://api.nvcf.nvidia.com/v2/nvcf/pexec/functions/0bf77f50-5c35-4488-8e7a-f49bb1974af6",
+            "ranking",
         ),
     ],
 )
 def test_registered_model_functional(
-    client: type, id: str, endpoint: str, contact_service: Any
+    client: type, id: str, endpoint: str, model_type: str, contact_service: Any
 ) -> None:
-    model = Model(id=id, endpoint=endpoint)
+    model = Model(
+        id=id, endpoint=endpoint, client=client.__name__, model_type=model_type
+    )
```

Comment:
@mattf I have updated this because register_model method does not set client and this was failing.
Is this a bug in register_model method?? As it should get set to client type?

Reply:
it's very good that you raise this. we have users for `register_model` and want to maintain api compatibility.

how can we do `register_model` without requiring users to change their code?

Reply:
I have reverted this test case and replaced error with warning